content/blog/2018-05-10-onboarding-social-weather.md

```diff
@@ -0,0 +1,698 @@
+                     line = stringr::str_replace_all(line, "`", ""),
+                     # only keep text from links, not the links themselves
+                     line = stringr::str_replace_all(line, "\\]\\(.*\\)", ""),
+                     line = stringr::str_replace_all(line, "\\[", ""),
+                     line = stringr::str_replace_all(line, "blob\\/master", ""), 
+                     # ’
+                     line = stringr::str_replace_all(line, "’", ""), 
+                     # remove some other links
+                     line = stringr::str_replace_all(line, "https\\:\\/\\/github\\.com\\/", ""))
+
+threads <- dplyr::filter(threads, !stringr::str_detect(line, "estimated hours spent reviewing"))
+threads <- dplyr::filter(threads, !stringr::str_detect(line, "notifications@github\\.com"))
+threads <- dplyr::filter(threads, !stringr::str_detect(line, "reply to this email directly, view it on"))
+threads <- dplyr::ungroup(threads)
+```
+
+Here is what we get from the same review.
+
+``` r
+dplyr::filter(threads, user == "jsta", is_review)  %>%
+  dplyr::select(line) %>%
+  head() %>%
+  knitr::kable()
+```
+
+<table>
+<colgroup>
+<col style="width: 100%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th style="text-align: left;">line</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td style="text-align: left;">* also, you might consider using the skip_on_cran function for lines that call an external download as recommended by the ropensci packaging guide.</td>
+</tr>
+<tr class="even">
+<td style="text-align: left;">* i am having timeout issues with building the getting_started vignette. i wonder if there is a particular year with very few hurricanes that would solve the timeout problem.</td>
+</tr>
+<tr class="odd">
+<td style="text-align: left;">* i cannot build the data.world vignette. probably because i don’t have an api key set up. you may want to consider setting the code chunks to eval=false.</td>
+</tr>
+<tr class="even">
+<td style="text-align: left;">* i really like the tidy_ functions. i wonder if it would make it easier on the end-user to have the get_ functions return tidy results by default with an optional argument to return “messy” results.</td>
+</tr>
+<tr class="odd">
+<td style="text-align: left;">* missing a maintainer field in the description</td>
+</tr>
+<tr class="even">
+<td style="text-align: left;">* there are no examples for knots_to_mph, mb_to_in, status_abbr_to_str, get_discus, get_fstadv, tidy_fstadv, tidy_wr, tidy_fcst. maybe some can be changed to non-exported?</td>
+</tr>
+</tbody>
+</table>
+
+<br><br>
+
+So now, we mostly got the interesting human and original language.
+
+This got me “tidy enough” text. Let’s not mention this package author
+who found a way to poorly format [their
+submission](https://github.com/ropensci/onboarding/issues/24) right
+under a guideline explaining how to copy the DESCRIPTION… Yep, that’s
+younger me. Oh well.
+
+#### Computing sentiment
```

Comment:
I was expecting some output or conclusion after this segment on sentiment - if it gets used later, mention that here